### PR TITLE
Blank slate is now clickable and opens the add WI dialog

### PR DESCRIPTION
**Pull request by michaelkleinhenz**
Blank slate is now clickable and opens the add WI dialog

### CODE REVIEW

File: src/app/work-item/work-item-list/work-item-list.component.scss [hunk 1]
```diff
@@ -135,6 +135,19 @@
             border: solid 0px red;
             position: relative;
             align-items: stretch;
+
+            .alm-blank-slate {
+                width: 90%;
+                display: block;
+                color: black;
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-list/work-item-list.component.scss]: change this to `#363636`

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1685-fabric8-planner.int.rdu2c.fabric8.io

**Comment by michaelkleinhenz** [comment]
@mindreeper2420 done.

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1685-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1685-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1685-fabric8-planner.int.rdu2c.fabric8.io

**Pull request merged**